csr/csr.go

```diff
@@ -50,6 +50,12 @@ func VerifyCSR(csr *x509.CertificateRequest, maxNames int, keyPolicy *goodkey.Ke
 	if err := csr.CheckSignature(); err != nil {
 		return errors.New("invalid signature on CSR")
 	}
+	if len(csr.EmailAddresses) > 0 {
+		return errors.New("CSR contains one or more email address fields")
```

Comment:
While this is the style that we are already using would you have any objection to making this a var so that you don't have to match it exactly in `csr_test.go`?